packages/editor/src/store/constants.js

```diff
@@ -13,6 +13,8 @@ export const EDIT_MERGE_PROPERTIES = new Set( [ 'meta' ] );
  */
 export const STORE_NAME = 'core/editor';
 
+export const EDIT_POST_STORE_NAME = 'core/edit-post';
```

Comment:
Is there a better way to do this? I can't import store from edit-post, I guess it's because edit-post imports editor store. Memory error when I tried.